src/logstash/scripts/production_locations/business_url.rb

```diff
@@ -0,0 +1,8 @@
+def filter(event)
+    business_url = event.get('business_url')
```

Comment:
Could you please use `business_url_value` as in other fields?

Reply:
done